docs/deploying-airbyte/local-deployment.md

```diff
@@ -28,5 +28,11 @@ Instead of cloning the repo, you can alternatively download the latest Airbyte r
 
 ## Troubleshooting
 
+**I have a MacBook with M1 chip, is it possible to run Airbyte?**
+Some users using MacBook with M1 chip are facing some problems running Airbyte.
+The problem is related with the chip and docker.
+There is some [documentation](https://docs.docker.com/docker-for-mac/apple-silicon/) on Docker website.
```

Comment:
I don't think we should point users to the Docker M1 docs since it's going to be a huge issue for them to deal with.
It's probably better to link to https://github.com/airbytehq/airbyte/issues/2017 and suggest they follow to see when it's available.

Reply:
Thanks @jrhizor didnt know about that issue.